refs/biblio.json

```
@@ -1935,6 +1935,11 @@
         "status": "W3C Incubator Group Report",
         "publisher": "W3C"
     },
+    "LNG-TAG": {
```

Maybe more readable?: 

```suggestion
    "LANG-SUBTAG-REGISTRY": {
```

---

Also less change it will be used to cite BCP47.